src/calendars/general.ts

```diff
@@ -392,6 +401,15 @@ const dates = async (config: RomcalConfig): Promise<Array<LiturgicalDayInput>> =
       date: dayjs.utc(`${year}-5-3`),
       liturgicalColors: LiturgicalColors.WHITE,
     },
+    {
+      key: 'john_of_avila_priest',
+      rank: Ranks.OPT_MEMORIAL,
+      date: dayjs.utc(`${year}-5-10`),
+      liturgicalColors: LiturgicalColors.WHITE,
+      metadata: {
+        titles: [Titles.MARTYR],
```

Comment:
Martyr and white? (same below in `spain.ts`)

Reply:
Oops, my mistake cause by copy-pasting without proofing. :wink:

Thanks! :pray: